tests/reg_tests/test_adjoint.py

```diff
@@ -54,7 +54,7 @@ def twist(val, geo):
     def span(val, geo):
         # Span
         C = geo.extractCoef("wing")
-        for i in range(len(C) - 1):
+        for _i in range(len(C) - 1):
```

Comment:
I have no idea whether the original code was correct since I've not used the span function before. Why do we not use the loop variable here?

Reply:
This looks like a bug to me, we should move all the `refAxis` points, the loop is clearly useless otherwise

Reply:
Yeah that was my thought also, but I don't really know how to fix it. In any case this highlights how checking unused loop variables can prevent bugs.

Reply:
@joanibal it looks like you were involved in #47 when we updated the testing section, do you recall how is this DV supposed to work?

Reply:
I agree that this is a bug. The span variable should be something like:
```
        def span(val, geo):

            C = geo.extractCoef("wing")
            s = geo.extractS("wing")

            for i in range(len(C)):
                C[i, 2] = C[i, 2] + s[i] * val[0]

            geo.restoreCoef(C, "wing")
```

However, I don't think this is affecting the test coverage and fixing it would require retraining the test. I can create an issue and then merge this in as is.

Reply:
Agreed @sseraj

Reply:
The issue is at #163

Reply:
I have no explanation for this. It was probably a temporary change for debugging that accidentally became permanent.